### PR TITLE
Fixed_navbar_at_top

### PR DESCRIPTION
**Pull request by raj-maurya**
fixed navbar at the top while scrolling the page issue [ #4 ].

Please review the PR.

Thank You!

### CODE REVIEW

**Comment by SaptakS** [comment]
@raj-maurya thanks. Can you please recheck the indentations and the newlines and white spaces? They seem a little out of place.

**Comment by raj-maurya** [comment]
@SaptakS 

Thanks for the suggestion. I have fixed the indentation,newlines and whitespaces.

Thank You!

File: index.html [hunk 1]
```diff
@@ -172,9 +172,10 @@ <h2 class="fs-2">College of Engineering, Pune, India</h2>
 
     </div><!-- responsive-page -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+      <script src="js/scroll.js"></script>
     <script src="https://unpkg.com/leaflet@1.0.1/dist/leaflet.js"></script>
     <script src="js/leaflet_sleep.js"></script>
-    <script src="js/parallax.js"></script>
+    <script src="js/parallax.js"></script>
```
Comment (vivekanand1101) [inline index.html:178]: extra whitespace ?

File: index.html [hunk 2]
```diff
@@ -172,9 +172,10 @@ <h2 class="fs-2">College of Engineering, Pune, India</h2>
 
     </div><!-- responsive-page -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+      <script src="js/scroll.js"></script>
```
Comment (vivekanand1101) [inline index.html:175]: indentation not correct

**Pull request closed without merging**